test/acceptance/test/install_tests.go

```diff
@@ -131,6 +131,79 @@ var _ = Describe("Weave GitOps Install Tests", func() {
 		})
 	})
 
+	It("Verify that gitops can install & uninstall gitops components to multiple clusters under a user-specified namespace via pull requests", func() {
+		namespace := "test-namespace"
+
+		_, cluster1Context, err := ResetOrCreateCluster(namespace, true)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		private := true
+		tip := generateTestInputs()
+		appRepoRemoteURL := "git@github.com:" + GITHUB_ORG + "/" + tip.appRepoName + ".git"
+
+		defer deleteRepo(tip.appRepoName, gitproviders.GitProviderGitHub, GITHUB_ORG)
+
+		By("And application repo does not already exist", func() {
+			deleteRepo(tip.appRepoName, gitproviders.GitProviderGitHub, GITHUB_ORG)
+		})
+
+		repoAbsolutePath := initAndCreateEmptyRepo(tip.appRepoName, gitproviders.GitProviderGitHub, private, GITHUB_ORG)
+
+		installAndVerifyWegoViaPullRequest(namespace, appRepoRemoteURL, repoAbsolutePath)
+
+		cluster2Name, cluster2Context, err := ResetOrCreateClusterWithName(namespace, false, "", true)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		defer func() {
+			selectCluster(cluster2Context)
+			deleteCluster(cluster2Name)
+			selectCluster(cluster1Context)
+		}()
+
+		selectCluster(cluster2Context)
+		installAndVerifyWegoViaPullRequest(namespace, appRepoRemoteURL, repoAbsolutePath)
+
+		selectCluster(cluster1Context)
+		cmd := fmt.Sprintf("%s uninstall --namespace %s --force", WEGO_BIN_PATH, namespace)
+		c := exec.Command("sh", "-c", cmd)
+		c.Stdout = os.Stdout
+		c.Stderr = os.Stderr
+		err = c.Start()
+
+		Expect(err).ShouldNot(HaveOccurred())
+
+		err = c.Wait()
+
+		Expect(err).ShouldNot(HaveOccurred())
+
+		_ = waitForNamespaceToTerminate(namespace, NAMESPACE_TERMINATE_TIMEOUT)
+
+		By("Then I should not see any gitops components", func() {
+			_, errOutput := runCommandAndReturnStringOutput("kubectl get ns " + namespace)
+			Eventually(errOutput).Should(ContainSubstring(`Error from server (NotFound): namespaces "` + namespace + `" not found`))
+		})
+
+		selectCluster(cluster2Context)
+
+		cmd = fmt.Sprintf("%s uninstall --namespace %s --force", WEGO_BIN_PATH, namespace)
+		c = exec.Command("sh", "-c", cmd)
+		c.Stdout = os.Stdout
+		c.Stderr = os.Stderr
+		err = c.Start()
+
+		Expect(err).ShouldNot(HaveOccurred())
+
+		err = c.Wait()
+		Expect(err).ShouldNot(HaveOccurred())
+
+		_ = waitForNamespaceToTerminate(namespace, NAMESPACE_TERMINATE_TIMEOUT)
+
+		By("Then I should not see any gitops components", func() {
+			_, errOutput := runCommandAndReturnStringOutput("kubectl get ns " + namespace)
+			Eventually(errOutput).Should(ContainSubstring(`Error from server (NotFound): namespaces "` + namespace + `" not found`))
```

Comment:
This block seems unnecessary.  Looking at the code for waitForNamespaceToTerminate() it polls k8s until the ns is removed or errors out.  Seems like we can just use that function with an `Expect(waitForNamespaceToTerminate(namespace, NAMESPACE_TERMINATE_TIMEOUT)).ShouldNot(HaveOccurred())` ?